src/ol_dbt/models/marts/combined/marts__combined_program_enrollment_detail.sql

```diff
@@ -193,27 +217,144 @@ with mitxpro__programenrollments as (
+    from mitx_programs
+    inner join mitx__courses
+        on mitx_programs.course_number = mitx__courses.course_number
+    where mitx__courses.is_on_mitxonline = false
+
+    union all
+
+    select
+        coalesce(mitx__courses.course_readable_id, edxorg_mitx_program_courses.course_readable_id) as course_readable_id
+        , null as program_readable_id
+    from edxorg_mitx_program_courses
+    left join mitx_programs
+        on edxorg_mitx_program_courses.program_name = mitx_programs.program_title
+    left join mitx__courses
+        on
+            replace(edxorg_mitx_program_courses.course_readable_id, 'MITx/', '')
+            = replace(mitx__courses.course_readable_id, 'course-v1:MITxT+', '')
+    where mitx_programs.program_title is null
+
+    union all
+
+    select
+        mitxpro__courses.course_readable_id
+        , mitxpro__programs.program_readable_id
+    from mitxpro__courses
+    inner join mitxpro__programs
+        on mitxpro__courses.program_id = mitxpro__programs.program_id
+)
+
+, final_combined_programs as (
+    select
+        combined_programs.platform_name
+        , combined_programs.program_id
+        , combined_programs.program_title
+        , combined_programs.program_name
+        , combined_programs.program_track
+        , combined_programs.program_type
+        , combined_programs.program_is_live
+        , combined_programs.program_readable_id
+        , {{ generate_hash_id('cast(combined_programs.user_id as varchar) || combined_programs.platform_name') }}
+        as user_hashed_id
+        , combined_programs.user_id
+        , combined_programs.user_email
+        , combined_programs.user_username
+        , combined_programs.user_full_name
+        , combined_programs.user_has_completed_program
+        , combined_programs.programenrollment_is_active
+        , combined_programs.programenrollment_created_on
+        , combined_programs.programenrollment_enrollment_status
+        , combined_programs.programcertificate_created_on
+        , combined_programs.programcertificate_is_revoked
+        , combined_programs.programcertificate_uuid
+        , combined_programs.programcertificate_url
+        , cast(substring(combined_programs.programcertificate_created_on, 1, 10) as date)
+        as programcertificate_created_on_date
+        , sum(case when upper(combined_enrollments.course_title) like '%CAPSTONE%' then 1 else 0 end)
+        as capstone_sum
+        , min(cast(substring(combined_courseruns.courserun_start_on, 1, 10) as date))
+        as first_courserun_start_on_date
+        , count(distinct combined_enrollments.course_readable_id) as unique_courses_taken_in_program
+    from combined_programs
+    left join courses_in_programs
+        on combined_programs.program_readable_id = courses_in_programs.program_readable_id
+    left join combined_enrollments
+        on
+            cast(combined_programs.user_id as varchar) = combined_enrollments.user_id
```

Comment:
```suggestion
            combined_programs.user_id = combined_enrollments.user_id
```
This line was throwing an error when I tried to build the model.
`Database Error in model marts__combined_program_enrollment_detail (models/marts/combined/marts__combined_program_enrollment_detail.sql)
  TrinoUserError(type=USER_ERROR, name=TYPE_MISMATCH, message="line 334:56: Cannot apply operator: varchar = bigint", query_id=20250522_214547_26950_suwmf)
  compiled Code at target/run/open_learning/models/marts/combined/marts__combined_program_enrollment_detail.sql`
  I checked on Starburst and I believe the user_id fields from both tables (int__mitxpro__programenrollments and int__combined__courserun_enrollments) are bigint, so you shouldn't need to cast it here. Unless my dev schema where I am testing is off from what's in production.